todo/src/components/AddTodoForm.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actionDispatchers from '../state/actionDispatchers';
+
+const AddTodoForm = (props) => {
+  const inputRef = React.createRef();
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    props.addTodo(inputRef.current.value);
+    inputRef.current.value = "";
+  }
+  return (
+    <form>
+      <input ref={inputRef} type="text" name="" id="" placeholder="Todo" />
+      <button type="submit" onClick={handleSubmit} >Create Todo</button>
+    </form>
+  )
+}
+
+export default connect(
+  undefined,
```

Comment:
Because you asked me about this, I went to check. From React-Redux docs is fine to pass undefined or null instead of mapStateToProps when you don't need it.